src/zenml/integrations/evidently/__init__.py

```diff
@@ -56,6 +56,7 @@ class EvidentlyIntegration(Integration):
     REQUIREMENTS = [
         "evidently>=0.4.16,<=0.4.22",
         "tenacity!=8.4.0",  # https://github.com/jd/tenacity/issues/471
+        "numpy<2.0.0",  # evidently is not compatible with NumPy 2.0
```

Comment:
Doesn't evidently have this built-in to their own set of dependencies? Why do we have to explicitly limit this here?

Reply:
the latest evidently probably supports this but the versions we have pinned here do not apparently pin numpy @bcdurak